receiver/redisreceiver/config.go

```diff
@@ -24,6 +24,9 @@ type config struct {
 	configmodels.ReceiverSettings `mapstructure:",squash"`
 	// The duration between Redis metric fetches.
 	CollectionInterval time.Duration `mapstructure:"collection_interval"`
+	// The logical name of the Redis server. This value will be added as a
+	// "service.name" Resource label.
+	ServiceName string `mapstructure:"service_name"`
```

Comment:
Should this be "service.name" for consistency? Also maybe leave a TODO to allow users to configure more things in the `Resource`. Something like:

```
   Resource map[string]AttributeValue 
```

Also can be done in a separate PR :)

Reply:
Agreed. Done.